addons/binding/org.openhab.binding.mcp23017/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,29 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: mcp23017 Binding
+Bundle-SymbolicName: org.openhab.binding.mcp23017;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.1.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .,
+ lib/pi4j-core.jar,
+ lib/pi4j-gpio-extension.jar
+Import-Package: 
+ com.google.common.collect;version="10.0.1",
+ org.apache.commons.lang;version="2.6.0",
+ org.eclipse.smarthome.config.core,
+ org.eclipse.smarthome.core.library.types,
+ org.eclipse.smarthome.core.thing,
+ org.eclipse.smarthome.core.thing.binding,
+ org.eclipse.smarthome.core.thing.binding.builder,
+ org.eclipse.smarthome.core.thing.type,
+ org.eclipse.smarthome.core.types,
+ org.openhab.binding.mcp23017,
+ org.openhab.binding.mcp23017.handler,
+ org.openhab.core.binding,
```

Comment:
Are you sure you need 'org.openhab.core.binding' and 'org.openhab.core.library.types' what are you using from them?